app/assets/src/components/views/discovery/discovery_sidebar.scss

```diff
@@ -94,4 +94,14 @@
     text-align: right;
     color: white;
   }
+
+  .showHide {
+    text-transform: uppercase;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    color: $primary-light;
+    font-size: 10px;
```

Comment:
Can you find a mixin in typography to use here instead of setting font-size by yourself?